src/screens/SignupScreen.tsx

```diff
@@ -306,5 +317,12 @@ const styles = StyleSheet.create({
     container: {
         flexGrow: 1,
         justifyContent: 'center'
+    },
+    imageContainer:{
+        position:'relative',
+        width: 100, 
+        height: 100,
+        alignSelf: 'center'
```

Comment:
Not needed since `Image` component accepts a `size` prop makes the image responsive.